### ISSUE TITLE
Suggestion - create a flex container class

### ISSUE THREAD

**Issue by jzhu01** [Modelsson/tea_cozy#1]
Seen here: https://github.com/Modelsson/tea_cozy/blob/master/resources/css/style.css#L12

Since this should is repeated often throughout your code, it might be useful to create a class styling and use several other codes that are repeated as well: 
https://github.com/Modelsson/tea_cozy/blob/master/resources/css/style.css#L56-L57 (perhaps these?)

**Comment by Modelsson** [comment]
Yes, I see that now. In the project Secret Agent I already made a separate "container" just for the width and margin: 0 auto. I'm slowly getting the hang of it.
